src/main/kotlin/no/nav/syfo/melding/api/MeldingApi.kt

```diff
@@ -1,27 +1,27 @@
-package no.nav.syfo.behandlerdialog.api
+package no.nav.syfo.melding.api
 
 import io.ktor.http.*
 import io.ktor.server.application.*
 import io.ktor.server.request.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
 import no.nav.syfo.application.api.VeilederTilgangskontrollPlugin
-import no.nav.syfo.behandlerdialog.MeldingTilBehandlerService
-import no.nav.syfo.behandlerdialog.domain.toMeldingTilBehandlerResponseDTO
+import no.nav.syfo.melding.meldingTilBehandler.MeldingTilBehandlerService
+import no.nav.syfo.melding.meldingTilBehandler.domain.toMeldingTilBehandlerResponseDTO
 import no.nav.syfo.client.veiledertilgang.VeilederTilgangskontrollClient
 import no.nav.syfo.domain.PersonIdent
 import no.nav.syfo.util.*
 
-const val behandlerdialogApiBasePath = "/api/internad/v1/behandlerdialog"
-const val behandlerdialogApiMeldingPath = "/melding"
+const val meldingApiBasePath = "/api/internad/v1/behandlerdialog"
+const val meldingApiMeldingPath = "/melding"
```

Comment:
Synes denne ble litt kjip da, dobbelt `melding`.. Noen forslag?

Reply:
Nå bruker vi jo fortsatt behandlerdialog i path, selv om vi kaller det MeldingApi her, men det er kanskje OK (api for meldinger i behandlerdialog) 🤷‍♂️ Kunne man i så fall fortsatt kalt det `behandlerdialogApiBasePath` og `behandlerdialogApiMeldingPath` ?

Reply:
Hva med `meldingApiBasePath = /api/internad/v1/melding`? Hvorfor trenger man en basepath som slutter på appnavnet, og så melding utenfor der?

Reply:
Tja, kunne nok bare hatt `/melding` også ja, da blir det likt for Get og Post også 🤔